serveradmin/settings.py

```diff
@@ -90,6 +91,13 @@
 # Examples: "http://media.lawrence.com/media/", "http://example.com/media/"
 MEDIA_URL = '/media/'
 
+# Add compressor to static file finders to allow compressing CSS/JS on the fly
```

Comment:
Have you also considered to use the offline compressing?

Reply:
Yes I discarded it because of the possible Caveats. Especially the first bullet point sounded dangerous to me:

- https://django-compressor.readthedocs.io/en/stable/scenarios/#id1

Reply:
Same as above. Quite a lot in my opinion and if it were only for making sure that users get the latest version when something changes (Browser Cache).

- https://github.com/innogames/serveradmin/blob/master/serveradmin/servershell/templates/servershell/index.html#L170

Reply:
Yeah, I saw that some javascript has dynamic values, so javascript will get generated during runtime. Due to CSP and general Good Practice we anyway should try and avoid such javascript generation. So, offline compression might come back later then.

Reply:
> Due to CSP and general Good Practice
I lack to imagine a concrete for an attack vector for dynamic Javscript generation. Do you have some reference for this ?